Assets/BossRoom/Scripts/Client/UI/Lobby/LobbyPanelUI.cs

```diff
@@ -0,0 +1,43 @@
+using BossRoom.Scripts.Shared.Infrastructure;
+using BossRoom.Scripts.Shared.Net.UnityServices.Lobbies;
+using TMPro;
+using UnityEngine;
+
+namespace BossRoom.Scripts.Client.UI
+{
+    /// <summary>
+    /// An individual Lobby in the list of avaialble lobbies
+    /// </summary>
+    public class LobbyPanelUI : ObserverBehaviour<LocalLobby>
```

Comment:
Same comment as previous, "panel" is a bit misleading here, as it references the implementation instead of what it actually represents.

Reply:
"button" instead?